ddtrace/propagation/http.py

```diff
@@ -988,6 +988,7 @@ def parent_call():
             span_context = non_active_span.context
 
         if hasattr(ddtrace, "tracer") and hasattr(ddtrace.tracer, "sample"):
```

Comment:
nit: do we need this check? ddtrace.tracer should always be defined and the tracer should always have a `sample` method

Reply:
Good question 🤔 Honestly don't have enough historical context to back it up, but I'm assuming it used to _not_ always be the case? I can ask about this offline to the author for this code.